app/javascript/controllers/state_updates_controller.js

```diff
@@ -38,7 +38,9 @@ export default class extends Controller {
   renderUpdates(data) {
     for (const [attribute, value] of Object.entries(data)) {
       const target = this.targets.find(attribute)
-      target.innerHTML = value
+      if (target) {
+        target.innerHTML = value
+      }
```

Comment:
without this check, i got a javascript error in the browser when i manually triggered deposit completion for testing.  this seemed like a harmless check, and didn't break any existing tests.

the errors i got without this fix were like:
<img width="1357" alt="Screen Shot 2021-02-02 at 12 22 34 AM" src="https://user-images.githubusercontent.com/7741604/106577991-e4007d80-64f3-11eb-9c94-390f0cadec62.png">
<img width="882" alt="Screen Shot 2021-02-02 at 12 51 32 AM" src="https://user-images.githubusercontent.com/7741604/106577999-e531aa80-64f3-11eb-9587-f133cc94012b.png">

Reply:
the one in the browser manifested when i ran `./bin/rake complete_deposits` while i had the dashboard open in my browser.  the test suite failure happened in the new `spec/features/purl_reservation_spec.rb` without this change.

Reply:
I think this error must be due to the page that is rendered not having an element with `data-state-updates-target="purl"`.

Reply:
@jmartin-sul :speech_balloon: 
> * I noticed that I couldn't edit the top level `work_type` once the work was created.  Not sure whether that's intentional, but that would seem to be a bit in tension with the lack of opportunity to pick a work type when reserving a PURL.
> * The styling on the modal doesn't quite match the mockups, and in particular, I couldn't get the "(You can update this later.)" explanation under the header to display on its own line if I put it with the header in the same `modal-header` `div`.  Maybe there's a better way to structure that?
> * Where the ticket description and the linked google doc diverged, I went with the ticket description, since it seemed to have been updated more recently (the doc said to use a default title, but the ticket had a mockup with a title field; the doc said the status should be reserved, but the ticket just said to register the work).
> 

Thanks for writing this up! I'd like to get @amyehodge's takes on those.